upload-chunked-file.js

```diff
@@ -0,0 +1,43 @@
+function chunkFileUpload(file, serverUrl, chunkSize) {
```

Comment:
Consider adding parameter and type checks. For instance, check if `file` has a valid size, `serverUrl` is a valid URL string, and `chunkSize` is a positive number.